Game/Game.cls

```diff
@@ -30,6 +30,15 @@ public with sharing class Game
     	return winner;
     }
 
+	public void fakeMethod(String blah){
+		String fakeVar = 'nope';
+
+		fakeVar = fakeVar;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Avoid idempotent operations (like assigning a variable to itself). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://app.codescan.io/coding_rules#rule_key=sf%3AIdempotentOperations)